File: Reto-1/Reto-1.js
let contador = 1
while (contador <= 100) {
    if (contador % 3 == 0 && contador % 5 == 0) {
        console.log("FizzBuzz");
        contador += 1
    } else if (contador % 3 == 0) {
        console.log("Fizz");
        contador +=1
    } else if (contador % 5 == 0) {
        console.log("Buzz");
        contador += 1
    } else {
        console.log(contador);
        contador += 1
    } 
}

File: Reto-1/Reto-1.py
contador = 1
while (contador <= 100): 
    if (contador % 3 == 0 and contador % 5 == 0):
        print("FizzBuzz")
        contador = contador + 1
    elif contador % 3 == 0:
        print("Fizz")
        contador = contador + 1
    elif contador % 5 == 0:
        print("Buzz")
        contador = contador + 1
    else:
        print(contador)
        contador = contador + 1